unit_test/sparse/Test_Sparse_spmv_bsr.hpp

```diff
@@ -369,10 +371,12 @@ void check_bsrm_times_mv(const char fOp[], scalar_t alpha, scalar_t beta,
     double error = 0.0, maxNorm = 0.0;
     for (int jc = 0; jc < nrhs; ++jc) {
       for (int ir = 0; ir < nRow; ++ir) {
-        error   = std::max(error, Kokkos::ArithTraits<scalar_t>::abs(
-                                    h_ycrs(ir, jc) - h_ybsr(ir, jc)));
-        maxNorm = std::max(maxNorm,
-                           Kokkos::ArithTraits<scalar_t>::abs(h_ycrs(ir, jc)));
+        error =
+            std::max<double>(error, double(Kokkos::ArithTraits<scalar_t>::abs(
```

Comment:
Instead of doing this you could go with:
```c++
using KATS = Kokkos::ArithTraits<scalar_t>;
scalar_t error = KATS::zero();

error = KATS::max(error, KATS::abs(h_ycrs(ir, jc) - h_ubsr(ir, jc)));
```

Reply:
Where is the KATS::max( arg a, arg b) defined?
I could not find it